itou/employee_record/tests/tests_models.py

```diff
@@ -274,6 +276,10 @@ class EmployeeRecordLifeCycleTest(TestCase):
     Note: employee records status is never changed manually
     """
 
+    @pytest.fixture(autouse=True)
+    def _set_faker_fixture(self, faker):
+        self.faker = faker
```

Comment:
pour ma curiosité, pourquoi on ne mettrait pas ça dans un mixin de `employee_record.testing` comme `FakerMixin` ? Sinon, sympa comme moyen d'injecter des fixtures sur une classe testcase :)

Reply:
La méthode d'injection viens de la doc : https://docs.pytest.org/en/7.1.x/how-to/unittest.html#using-autouse-fixtures-and-accessing-other-fixtures :).
Pour le mixin j'ai hésité mais je me suis trouver réticent à l'idée d'introduire de l'héritage pour les `TestCase` alors qu'on veux plutôt transitionner vers du full pytest, mais en relisant la doc il y a sans doute une alternative pour avoir que les avantages ;).